guides/common/modules/con_registering-hosts-by-using-registration-templates.adoc

```diff
@@ -0,0 +1,15 @@
+[id="Registering_Hosts_{context}"]
+= Registering Hosts by Using Registration Templates
```

Comment:
I vote for unifying the name for the feature as `Global registration`

I see that sometimes we say `Global registration`, sometimes 'Registration templates`, might be confusing for the users